pygfx/renderers/wgpu/wgsl/mesh.wgsl

```diff
@@ -340,22 +340,22 @@ fn fs_main(varyings: Varyings, @builtin(front_facing) is_front: bool) -> Fragmen
     $$ endif
 
 
+    {$ include 'pygfx.uv_fragment.wgsl' $}
+
     $$ if color_mode == 'normal'
         var diffuse_color = vec4<f32>((normalize(surface_normal) * 0.5 + 0.5), 1.0);
     $$ else
         // to support color modes
         var diffuse_color = u_material.color;
-        $$ if colorspace == 'srgb'
-            diffuse_color = vec4f(srgb2physical(diffuse_color.rgb), diffuse_color.a);
-        $$ endif
+        diffuse_color = vec4f(srgb2physical(diffuse_color.rgb), diffuse_color.a);
```

Comment:
What's the purpose of this change?

Reply:
Uniform color is `gfx.Color`, not related to the colorspace of TexturemMap, and `gfx.Color` is all in the srgb space now.